script.js

```diff
@@ -1,96 +1,211 @@
-const playerContainer = document.getElementById('all-players-container');
-const newPlayerFormContainer = document.getElementById('new-player-form');
+const playerContainer = document.querySelector("#all-players-container");
+const newPlayerFormContainer = document.getElementById("new-player-form");
 
 // Add your cohort name to the cohortName variable below, replacing the 'COHORT-NAME' placeholder
-const cohortName = 'YOUR COHORT NAME HERE';
+const cohortName = "2302-acc-pt-web-pt-b";
```

Comment:
const should be all uppercase COHORTNAME

Reply:
Thank you. I'll capitalize them.